### ISSUE TITLE
Boot scripts

### ISSUE THREAD

**Issue by jaagut** [bit-bots/bitbots_misc#158]
@ftsell 
@texhnolyze 
@timonegk 

How do we want to handle all these bootup scripts to automatically start our software? I believe they are outdated. Do we even want to keep them?

Example:
https://github.com/bit-bots/bitbots_misc/blob/master/bitbots_utils/scripts/boot-highlevel.sh

**Comment by timonegk** [comment]
Yes, they are definitely outdated, we can remove them from the repo, but I would prefer to leave them there lest we forget how they worked. If I remember correctly, it was quite some effort to set them up correctly. I could especially imagine that we want to bring them back in some way for the competition.
